include/gridtools/common/hymap.hpp

```diff
@@ -138,6 +138,25 @@ namespace gridtools {
             };
             using type = keys;
         };
+
+        template <class HyMap>
+        GT_META_DEFINE_ALIAS(to_meta_map,
+            meta::zip,
+            (GT_META_CALL(meta::rename, (meta::list, GT_META_CALL(get_keys, HyMap))),
```

Comment:
Should I keep the rename here now?

Reply:
don't use `meta::zip` here. use `meta::transform`:
```
template <class HyMap>
GT_META_DEFINE_ALIAS(to_meta_map,
  meta::transform,
  (meta::list, GT_META_CALL(get_keys, HyMap), GT_META_CALL(tuple_util::traits::to_types, HyMap)));
```

Reply:
But isn’t this exactly equivalent to
```
template <class HyMap>
GT_META_DEFINE_ALIAS(to_meta_map,
  meta::zip,
  (GT_META_CALL(get_keys, HyMap),
        GT_META_CALL(tuple_util::traits::to_types, HyMap)));
```
apart from the higher efficiency gained by the template specializations available for `transform<F, L1<T1s...>, L2<T2s...>>`? I.e. shouldn’t we rather provide specializations for zip to make it as efficient as transform here as it’s not really logical that the transform version is better here for the average meta user?

Reply:
sorry. my bad. you are right.